server/src/main/java/org/folio/metastorage/server/OaiService.java

```diff
@@ -30,12 +30,13 @@ public final class OaiService {
 
   private OaiService() { }
 
-  static final String OAI_HEADER =
-      "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
-          + "<OAI-PMH xmlns=\"http://www.openarchives.org/OAI/2.0/\"\n"
-          + "         xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n"
-          + "         xsi:schemaLocation=\"http://www.openarchives.org/OAI/2.0/\n"
-          + "         http://www.openarchives.org/OAI/2.0/OAI-PMH.xsd\">\n";
+  static final String OAI_HEADER = """
+  <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You could increase the indentation, the minimum indent is stripped.

Reply:
I could.. And I would most certainly do it if "auto-indent" for my editor does that. But it doesn't . Visual Studio code?

Reply:
#43 .. actually OAI-PMH and xml header should be on same level 0 (without leading spaces)